catalog-info.yaml

```diff
@@ -194,6 +202,10 @@ spec:
         trigger_mode: "none"
       repository: "elastic/connectors"
       schedules:
+        Daily 8_16:
```

Comment:
@seanstory there is a comment at the top of this cron schedule that says it applies to versions < 8.16. Let me know if adding 8.16 here is not needed.

Reply:
Correct, no need to add this block. Once we ship 8.15.3 in a few days, we'll be able to delete this whole job spec.